### DISCUSSION TITLE
LinkIt Smart 7688 Duo | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#208] [Q&A]
# LinkIt Smart 7688 Duo | Seeed Studio Wiki

LinkIt Smart 7688 Duo

https://wiki.seeedstudio.com/LinkIt_Smart_7688_Duo/

**Comment by Laihu08** [comment]
Below link is not working to install board for linkit smart 7688
http://download.labs.mediatek.com/package_mtk_linkit_smart_7688_test_index.json

**Reply by MatthewJeffson** [reply]
Hi, maybe you can try with this: http://download.labs.mediatek.com/package_mtk_linkit_smart_7688_index.json

**Comment by Laihu08** [comment]
Hi, thank you for your reply, I tried your link still getting the below
error. moreover, when i click this link i cannot view the page, i think
this link is broken.


Some indexes could not be updated. Get "
http://download.labs.mediatek.com/package_mtk_linkit_smart_7688_index.json":
dial tcp: lookup download.labs.mediatek.com: no such host:
http://download.labs.mediatek.com/package_mtk_linkit_smart_7688_index.json

Thanks.
